modules/ROOT/pages/setup.adoc

```diff
@@ -0,0 +1,196 @@
+** Click the *New* button.
+** In the modal window, click the *choose file* button.
+** Upload the private certificate belonging to your organization.
+
+. Connect to Payload Storage. Anypoint Partner Manager enables you to connect your own storage (such as a database, Amazon S3, Azure Blob Storage, and so on) to store the incoming B2B message payload. To connect to your storage, establish an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage. +
+** Download the pre-packaged API and integration implementation from Anypoint Exchange.
+** Deploy the Mule app on your runtime.
+** Make a note of the API address configurations (host, port, path, and so on) that you will use later to associate the storage within Anypoint Partner Manager.
+
+_**Note to Maria** - Please obtain the link from Anypoint Exchange for the pre-packaged API and integration app from Manik / Pablo_
+
+. Once the API is setup, navigate back to your organization's profile and navigate to the **Payload Storage** section. +
+
+** Click the *Edit* button.
+** Provide the following details based on the API configuration you setup earlier - Host, Port, and Path. Per your needs, you can modify the default values provided for the Connection timeout and Response timeout.
+** Configure the authentication needed to access the API. The options include - Non, Basic Authentication, OAuth Token, Client Credentials, and API Key.
+
+All other configurations on the your organization's profile are optional at this point. Many of these details are populated as you setup the Message flows later on.
+
+=== Add and Configure Your Partners
+
+Typically partners provide the specific settings required to connect to their Endpoints. Just as with your organization's profile, most of the settings for your partners are optional. In most cases you only need to configure the following fields for each partner:
+
+** Partner Profile
+** Identifiers
+** Partner Certificates
+
+==== Add a New Partner
+
+. To add a new partner, select Partners from the left-hand navigation and click  *New Partner*. +
+. Add the 'Partner Name' and at least one identifier and click *Create Partner*. This is all you need to create a new partner. +
+
+==== Edit a Partner's Profile
+
+To edit a partner's profile, navigate to the *Partners* tab on the upper left. From the page for the specific Partner, you can add detailed information related to the partner. The information that can be made available here includes:
+
+** The Partner's Name.
+** A description of the Partner.
+** A Logo for the Partner (.png or .svg files are supported).
+
+_**Note to Maria** - Please add an image of the Partners details here_[Discussed the possibility of not using this ss with Amit.]
```

Comment:
Please be sure to search for any comment and add // to line start, that hides the content from the actual view.

Reply:
Got it, thanks ever so much.